libraries/grpc-sdk/src/index.ts

```diff
@@ -219,20 +224,36 @@ export default class ConduitGrpcSdk {
     this.config.watchModules().then();
     emitter.on('serving-modules-update', (modules: any) => {
       Object.keys(this._modules).forEach((r) => {
```

Comment:
Can we look into merging these 2 for loops? They seem to be mergeable, and I definitely don't remember why I had this split.

Reply:
Config won't send over any removed modules, as such we would still have to diff the arrays and `closeConnection()` on any removed modules.